### DISCUSSION TITLE
How to adjust php.ini file in Docker images?

### DISCUSSION THREAD

**Discussion by troycarpenter** [ampache/ampache#3360] [Q&A]
I've recently started to allow users to upload songs.  It looks like the php.ini file for docker images limits the uploads to 2MB which is WAY to small even for MP3 files.  I've made changes for myself by passing in a new php.ini file via docker-compose.  Is that the only way to do it?  I want to let a song be up to 15MB (to start).  To do that, requires changing both post_max_size and upload_max_filesize, but it would be nicer if that were in some environment variable so that people can set their own limits.

**Comment by Andrew-McGee** [comment]
I think that's the default for any php install. I don't use docker and I also had to update these settings in my php.ini. I set them both to 100M as I only use flac.

I noticed there is a docker-setup.sh script for setting permissions. You could probably include a prompt in that script that makes ini file edits at install time.

**Comment by lachlan-00** [comment]
I've added it to the repo with [af48357](https://github.com/ampache/ampache-docker/commit/af483573a3ed449da8767eba0cb74a5024026bb5)

That way you can maintain your own file on build.

I've set the sizes to 24M for upload and 32 post_max_filesize
